src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -43,6 +38,11 @@ export class TaskListBuilder {
       )
     }
 
+    return new TaskList(1, 'Before you start', tasks)
+  }
+
+  static buildRespondToClaimSection (draft: ResponseDraft, externalId: string): TaskList {
```

Comment:
isn't externalId on the draft object so you wouldn't need to pass both?
REQ

Reply:
No, external ID is on the claim draft object, here we have response draft.

Reply:
the base draft class has external id, is it undefined on response draft?

Reply:
Yes, it is undefined.